src/components/Book.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Book = function () {
+  return (
+    <div className="book-item">
+      <div className="book-desc">
+        <h5>Genre</h5>
+        <h2>Title</h2>
+        <p>Author</p>
+        <div>
+          <a href="http://localhost:3000/">Comments</a>
+          <a href="http://localhost:3000/">Remove</a>
+          <a href="http://localhost:3000/">Edit</a>
+        </div>
+      </div>
+      <div className="progress" />
```

Comment:
- [ ] The Book component is expected to take the book data as a prop and render the title and the author. Kindly fix it.